src/components/CreateArticles.jsx

```diff
@@ -15,7 +15,7 @@ import CloudinaryWidget from './common/CloudinaryWidget';
 function CreateArticleContainer(props) {
   /** passing global state loading as props into variables */
   const {
-    message, isLoading, token,
+    message, isLoading, token
```

Comment:
#### Nitpick
- So inside where you have created the `request` method, define the token there, by getting it from the sessionStorage. Let me know if you are having challenges implementing that.

Reply:
@aanchirinah This has been implemented. Token is saved in the session storage when a new login action is to be dispatched. Token is gotten and used as part of the request object in a single axios instance.